src/platform/tls_schannel.c

```diff
@@ -1793,6 +1785,9 @@ CxPlatTlsWriteDataToSchannel(
         ISC_REQ_CONFIDENTIALITY |
         ISC_RET_EXTENDED_ERROR |
         ISC_REQ_STREAM;
+    if (TlsContext->IsServer) {
+        ContextReq |= ASC_REQ_MUTUAL_AUTH;
+    }
```

Comment:
Don't we need some kind of config option to indicate that we require a client cert?